ap2-receiver.py

```diff
@@ -846,6 +846,16 @@ def upgrade_to_encrypted(self, client_address, shared_key):
                 exit(-1)
     except Exception:
```

Comment:
Why wait on an Exception? Make it a utility function, then add an optional argument to display it as standalone. Should make things easier

Reply:
@glmnet can you separate this into a different commit (and optionally, PR)? Then we can focus on just the actual changes regarding the subject